src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerContentFacetImpl.java

```diff
@@ -185,13 +189,16 @@ protected Content doPutContent(final String path,
 
     Asset asset = getOrCreateAsset(path, assetKind, group, name, version);
```

Comment:
> Although not a change in this PR. This method also creates a component. Noting that here because someone could incorrectly use the getOrCreateAsset that does not create a component

That's a good point, I'll break it out as a separate commit to rename those once I get this merged in.